resources/assets/css/_forms.css

```diff
@@ -263,3 +263,16 @@
 .disabled-tags-input:not(.disabled-tags-input-add) .taggle > .close {
     @apply hidden;
 }
+
+/* Disable arrows from numeric input */
+/* Chrome, Safari, Edge, Opera */
+.disabled-arrows-input input::-webkit-outer-spin-button,
```

Comment:
it's not `disabled`, more hidden 🤔 

`.hide-number-input-arrows` or something?

Reply:
Yeah agreed, changing